src/style/fims/_users.scss

```diff
@@ -0,0 +1,78 @@
+// Profile
+.profile {
+  .heading {
+    margin-top: 10px;
+    min-height: 0;
+
+    .username {
+      margin-top: 0px;
+
+      .fa {
+        margin-right: 10px;
+      }
+    }
+  }
+
+  .settings {
+    margin-top: 35px;
+    border-top: 1px solid $gray-lighter;
+
+    .section {
+      border-bottom: 1px solid $gray-lighter;
+
+      .title {
+        font-weight: 500;
+        margin-bottom: $line-height-computed;
+      }
+    }
+  }
+}
+
+.change-password-form {
+  display: flex;
+
+  .form-group {
+    margin-bottom: $form-group-margin-bottom;
+    margin-right: 15px;
+    position: relative;
+    .btn {
+      margin-top: 20px;
+    }
+  }
+
+  md-icon.password-indicator {
+    position: absolute;
+    top: 28px;
+    right: 5px;
+    font-size: 18px;
+  }
+
+  .control-label {
+    display: block;
+  }
+}
+
+// Reset Password Page
+.reset-pass-form {
+  md-icon.password-indicator {
+    position: absolute;
+    top: 10px;
+    right: 20px;
+    width: 18px;
+    min-width: 18px;
+    height: 18px;
+    min-height: 18px;
+    font-size: 18px;
+    line-height: 18px;
+  }
+}
+
+// Both Password Indicators
+.weak-password {
+  @extend .password-indicator;
```

Comment:
just fyi, I'm removing this line b/c it doesn't do anything. There is no `.password-indicator` class anymore ;)